src/elements/content-sidebar/activity-feed/task-new/Task.js

```diff
@@ -41,25 +41,26 @@ class Task extends React.Component<Props> {
             currentUser,
             due_at,
             error,
+            getAvatarUrl,
+            getMentionWithQuery,
+            getUserProfileUrl,
             id,
             isPending,
+            mentionSelectorContacts,
+            name,
+            onAssignmentUpdate = noop,
             onDelete,
             onEdit,
-            onAssignmentUpdate = noop,
-            name,
+            permissions,
             status,
             translatedTaggedMessage,
             translations,
-            getAvatarUrl,
-            getUserProfileUrl,
-            getMentionWithQuery,
-            mentionSelectorContacts,
         } = this.props;
 
-        // Enable when edit/delete are implemented for new tasks
         const taskPermissions = {
+            ...permissions,
+            // can_edit: permissions.can_update,
```

Comment:
We'd prefer to avoid commented code being committed.

Reply:
Will remove in follow-up implementation of edit function